### PR TITLE
ClassLink integration

### PR DESCRIPTION
**Pull request by Emily-ejag**
ClassLink Integration needs to be linked to firekit branch enh/classlink-sso

Sibling PRs:
- [Firekit](https://github.com/yeatmanlab/roar-firekit/pull/103)
- [Cloud Functions](https://github.com/yeatmanlab/roar-firebase-functions/pull/225)

### CODE REVIEW

**Review by richford — APPROVED** [review]

File: src/components/auth/AuthClassLink.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+const props = defineProps({
+  code: { type: String, required: true },
+});
+
+const router = useRouter();
+const authStore = useAuthStore();
+const { isFirekitInit } = storeToRefs(authStore);
+
+console.log('in AuthClassLink component');
```
Comment (lucasxsong) [inline src/components/auth/AuthClassLink.vue:18]: should this console.log be removed?

File: src/router/index.js [hunk 2]
```diff
@@ -244,6 +250,14 @@ const routes = [
     props: (route) => ({ code: route.query.code }),
     meta: { pageTitle: 'Clever Authentication' },
   },
+  {
+    path: '/auth-classlink',
```
Comment (lucasxsong) [inline src/router/index.js:254]: Can I suggest that this path be edited to `/classlink-auth` for consistency?

**Review by lucasxsong — APPROVED** [review]

File: src/router/index.js (hunk 2 above)
Comment (Emily-ejag) [reply src/router/index.js:254]: I think auth-classlink is consistent with auth-clever and auth-email-link

File: src/components/auth/AuthClassLink.vue (hunk 1 above)
Comment (Emily-ejag) [reply src/components/auth/AuthClassLink.vue:18]: yes, thank you

**Pull request merged**